src/core/tests/Test_StatusHandlerTruncation.py

```diff
@@ -0,0 +1,1077 @@
+# Copyright 2023 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.7+
+import glob
+import json
+import os
+import random
+import sys
+import string
+import tempfile
+import time
+import unittest
+from core.src.bootstrap.Constants import Constants
+from core.tests.library.ArgumentComposer import ArgumentComposer
+from core.tests.library.RuntimeCompositor import RuntimeCompositor
+
+
+class TestStatusHandlerTruncation(unittest.TestCase):
+    def setUp(self):
+        self.runtime = RuntimeCompositor(ArgumentComposer().get_composed_arguments(), True)
+        self.container = self.runtime.container
+
+    def tearDown(self):
+        self.runtime.stop()
+
+    def __mock_os_remove(self, file_to_remove):
+        raise Exception("File could not be deleted")
+
+    def test_if_status_file_resets_on_load_if_malformed(self):
```

Comment:
This was a pre-existing test, which does not validate truncation logic and should not be in this file. Please move it back to it's original state.

Not marking every other similar test case here, but please make sure they are reverted as well

Reply:
move all these utests back to test_statushandler.py, no changes to variables and no additional description, 
done.